internal/corazawaf/transaction.go

```diff
@@ -1591,6 +1609,20 @@ func (tx *Transaction) generateResponseBodyError(err error) {
 	tx.variables.resBodyProcessorErrorMsg.Set(err.Error())
 }
 
+// setTimeVariables generates all the time variables
+func (tx *Transaction) setTimeVariables() {
+	timestamp := time.Unix(0, tx.Timestamp)
+	tx.variables.time.Set(timestamp.Format(time.TimeOnly))
+	tx.variables.timeDay.Set(strconv.Itoa(timestamp.Day()))
+	tx.variables.timeEpoch.Set(strconv.FormatInt(timestamp.Unix(), 10))
+	tx.variables.timeHour.Set(strconv.Itoa(timestamp.Hour()))
```

Comment:
I am inclined to say we can come up with a map for this instead of doing the conversion since these values are deterministic. Still worth to write a benchmark

Reply:
Do you propose to use a `TIME_MAP` variable with keys such as `hour`, `day`, `month`, `year`, and so on? Did I get this right? 

If so, this approach would further widen the compatibility gap between ModSecurity and Coraza, whereas the main goal of this PR is to reduce it.

Reply:
Sorry if I wasn't clear enough. I was advocating for having a map in memory e.g.
```go
var hourConversion map[int]string = map[int]string{1: "1", 2: "2", ...}
```

instead of doing the `strconv.Itoa`.

Reply:
Speaking of benchmarking, I run BenchmarkTransactionCreation - because all the values are set in the constructor.

Before changes:
```
cpu: Intel(R) Core(TM) i5-10210U CPU @ 1.60GHz
BenchmarkTransactionCreation
BenchmarkTransactionCreation-8   	   53013	     21910 ns/op
PASS
```

After changes:
```
BenchmarkTransactionCreation
BenchmarkTransactionCreation-8   	   51748	     22462 ns/op
PASS
```

However, the results vary widely from run to run. Am I missing something? Maybe, you could explain in more detail, what should I benchmark?